nservicebus/messaging/message-correlation.md

```diff
@@ -7,14 +7,18 @@ related:
  - nservicebus/messaging/headers
 ---
 
-Message correlation is the act of connecting instances of request messages with instances of response messages. Since all messages have a `Message Id` the `Correlation Id` is just a reference back to the specific message instance that caused this message to be sent back as a reply.
+Message correlation connects instances of request messages with instances of response messages. Each message has a `Message Id` and the `Correlation Id` is a reference back to the specific message instance that caused this message to be sent back as a reply.
```

Comment:
>How's that different from NServiceBus.RelatedTo?

RelatedTo is a pointer to the message that caused a given message to be sent. Ie if you process `PlaceOrderCommand` and as a result publish the `OrderPlacedEvent`, the event will have:

RelatedTo= PlaceOrderCommand.MessageId

This allows us , together with the conversationId to create the message flow diagram in SI.

Does this make sense?